test/classification.jl

```diff
@@ -75,6 +75,17 @@ for (fun, mask) = ((true_positives,  (0,0,0,1)),
     end
 end
 
+# f_score
+
+y_true = [0, 0, 0, 1, 1, 1, 1, 1, 2, 2, 2]
+y_pred = [0, 1, 0, 1, 0, 1, 0, 1, 2, 2, 1]
+@test f_score(y_true, y_pred) == Dict{Int, Float64}(0 => 0.5714285714285715, 1 => 0.6, 2=>0.8)
```

Comment:
could you box the `f_score(...)` into `@inferred(f_score(...))` to test that they are type stable?

Reply:
There seems to be a code stability issue in `LabelEnc.NativeLabels`. I wasn't able to workout a solution to fix it.

Here is an example of the issue.
```
julia> using MLLabelUtils
julia> @code_warntype LabelEnc.NativeLabels([0,1,2])
Variables:
  #self#::Any
  label::Array{Int64,1}

Body:
  begin 
      return (MLLabelUtils.LabelEnc.NativeLabels)(label::Array{Int64,1}, (Core.apply_type)(MLLabelUtils.LabelEnc.Val, (Base.arraylen)(label::Array{Int64,1})::Int64)::Type{Val{_}} where _)::MLLabelUtils.LabelEnc.NativeLabels{Int64,_} where _
  end::MLLabelUtils.LabelEnc.NativeLabels{Int64,_} where _
```

Reply:
Right. That's expected. Does it poison return type inference? If so we could fix that by annotating the return value for `f_score`. 

But this isn't high priority. I could do that some other time. Maybe just box those lines into `@inferred`, where `@inferred(f_score(...))` works right now.